packages/telemed-ehr/app/src/telemed/features/appointment/MedicalHistoryTab/AdditionalQuestions/AdditionalQuestionsPatientColumn.tsx

```diff
@@ -24,69 +17,44 @@ export const AdditionalQuestionsPatientColumn: FC = () => {
-        {isAppointmentLoading ? (
-          <Skeleton>
-            <Typography>Yes</Typography>
-          </Skeleton>
-        ) : (
-          <Typography>{fluVaccine}</Typography>
-        )}
-      </Box>
-      <Divider />
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'space-between',
-        }}
-      >
-        <Typography>Are your or your child&apos;s vaccines up to date?</Typography>
-        {isAppointmentLoading ? (
-          <Skeleton>
-            <Typography>Yes</Typography>
-          </Skeleton>
-        ) : (
-          <Typography>{vaccinesUpToDate}</Typography>
-        )}
-      </Box>
-      <Divider />
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'space-between',
-        }}
-      >
-        <Typography>Have you traveled out of the USA in the last 2 weeks?</Typography>
-        {isAppointmentLoading ? (
-          <Skeleton>
-            <Typography>Yes</Typography>
-          </Skeleton>
-        ) : (
-          <Typography>{travelUsa}</Typography>
-        )}
-      </Box>
-      <Divider />
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'space-between',
-        }}
-      >
-        <Typography>Has the patient been hospitalized in the past 6 months?</Typography>
-        {isAppointmentLoading ? (
-          <Skeleton>
-            <Typography>Yes</Typography>
-          </Skeleton>
-        ) : (
-          <Typography>{hospitalize}</Typography>
-        )}
-      </Box>
+      {isAppointmentLoading ? (
+        <Skeleton />
+      ) : (
+        Array.from(additionalQuestionsKeyMap.keys()).map((linkId) => {
+          const question = questionnaireResponse?.item?.find((q) => q.linkId === linkId);
+
+          if (!question) {
+            return null;
+          }
+
+          const answer = question?.answer?.find((answer) => answer?.valueString)?.valueString;
+          return answer ? (
+            <>
+              <Divider />
+              <Typography variant="body2" sx={{ color: 'grey' }}>
+                {additionalQuestionsKeyMap.get(question.linkId)}
+              </Typography>
+              <Typography>{answer}</Typography>
+            </>
+          ) : null;
+        })
+      )}
     </Box>
   );
 };
+
+const additionalQuestionsKeyMap = new Map([
+  ['rnn-time-zone-answer', 'Timezone'],
+  ['flu-vaccine', 'Have you or your child had your flu vaccine?'],
+  ['vaccines-up-to-date', "Are your or your child's vaccines up to date?"],
+  ['travel-usa', 'Have you traveled out of the USA in the last 2 weeks?'],
+  ['hospitalize', 'Has the patient been hospitalized in the past 6 months?'],
+  ['rnn-same-room-answer', 'Does your child sleep in their own room?'],
+  ['rnn-bottle-or-breastfeed-answer', 'Does your child currently bottle feed, breastfeed?'],
+  ['rnn-sleep-concerns-information', 'Do you have any sleep concerns?'],
+  ['rnn-sleep-goals-information', 'Do you have any sleep goals?'],
```

Comment:
Can we get the question text from the questionnaire? Otherwise we are maintaining text in two places.